packages/create-astro/src/index.ts

```diff
@@ -338,25 +265,10 @@ export async function main() {
 			name: 'typescript',
 			message: 'How would you like to setup TypeScript?',
 			choices: [
-				{
-					title: 'Relaxed',
-					value: 'base',
-				},
-				{
-					title: 'Strict (recommended)',
-					description: 'Enable `strict` typechecking rules',
-					value: 'strict',
-				},
-				{
-					title: 'Strictest',
-					description: 'Enable all typechecking rules',
-					value: 'strictest',
-				},
-				{
-					title: 'I prefer not to use TypeScript',
-					description: `That's cool too!`,
-					value: 'optout',
-				},
+				{ value: 'strict', title: 'Strict', description: '(recommended)' },
+				{ value: 'strictest', title: 'Strictest' },
+				{ value: 'base', title: 'Relaxed' },
+				{ value: 'unsure', title: `Hmm... I'm not sure` },
```

Comment:
One of the issues we heard here was:

- user picks the "idk" option
- we output a bunch of stuff to the terminal explaining what is up
- final output comes right afterwards, so they never read this message

nit: Thoughts on something like "Help me pick" here instead which makes it clear that we will make a decision for you? If not, I think it's okay as is but would remove the `Hmm...`  as unnecessary.

Reply:
Or, I still like my original idea that selecting this could output the helpful message, and then re-output the prompt, forcing them to read the message, understand that Astro users TypeScript, and then choose "Relaxed" themselves. 

More steps, but more reassuring than "I clicked idk and then a bunch of stuff happened and the CLI exited and I think I'm okay?"